docs/atm10/oredistro.md

```diff
@@ -1,73 +1,79 @@
 ---
 title: Mining Dimension Ore Distribution
 description: All ore spawn levels in the Mining Dimension!
-authors: 
- - Vidal (Original)
- - ArcTrooper (Editor)
+authors:
+    - Vidal (Original)
+    - ArcTrooper (Editor)
 ---
 
 # Mining Dimension Ore Distribution
 
 [Vidal's Original Spreadsheet.](https://docs.google.com/spreadsheets/d/1WA9Ek1k17u6YA9JJBSayUL1x_D8OMp134oQeVXBftDU/edit?gid=0#gid=0)
 
 ???+ warning "Max/Min Y-Level Sort Broken"
-    The sorting for the y-levels is broken for numbers currently. Alphabetical sorting is the only one that works right now. I will fix sorting by numbers in the future.
+The sorting for the y-levels is broken for numbers currently. Alphabetical sorting is the only one that works right now. I will fix sorting by numbers in the future.
+
+## Ore Distribution Visualization
+
+![Mining Dimension Ore Distribution](./img/atm10_mining_dim_ore_distribution.jpeg)
```

Comment:
Needs to be a double period before the `/img` folder

Reply:
Why would this be a double period? the `img` folder is a peer of the `oredistro.md` file.  If i try to use a relative path to go up to parent and over, it breaks.  

It would either be no dots at all or one dot.

### image folder parent as you suggest.
<img width="1128" alt="image" src="https://github.com/user-attachments/assets/33400b29-63e2-463a-9f9c-bdea53e0d47f">

### relative folder sibling
<img width="1184" alt="image" src="https://github.com/user-attachments/assets/927acb4a-162f-47f3-8f8d-8b573751cadc">

Reply:
> Why would this be a double period? the `img` folder is a peer of the `oredistro.md` file.  If i try to use a relative path to go up to parent and over, it breaks.  
> 
> It would either be no dots at all or one dot.
> 
> ### image folder parent as you suggest.
> <img width="1128" alt="image" src="https://github.com/user-attachments/assets/33400b29-63e2-463a-9f9c-bdea53e0d47f">
> 
> ### relative folder sibling
> <img width="1184" alt="image" src="https://github.com/user-attachments/assets/927acb4a-162f-47f3-8f8d-8b573751cadc">
> 
> 

Ah, I forgot the file structure when i reviewed this, my bad. But now I see you don't need `./` at all you can just do `img/pathName`